creme/creme_core/models/auth.py

```diff
@@ -1273,6 +1307,12 @@ def has_perm_to_view_or_die(self, entity: 'CremeEntity') -> None:
                 )
             )
 
+    def refresh_from_db(self, *args, **kwargs):
+        self._teams = None
+        self._teammates = None
+        self._settings = None
+        super().refresh_from_db(*args, **kwargs)
+
```

Comment:
Unit test (I presume it's better to test that queries are done with assertNumQueries() than testing protected attributes).